src/Containers/FileUpload/BottomPanel.tsx

```diff
@@ -0,0 +1,13 @@
+import React,{forwardRef} from 'react'
+import {Container,IContainerProps} from './Container'
+
+interface IBottomPanelProps extends IContainerProps{
+}
+
+export const BottomPanel=forwardRef<HTMLDivElement,IBottomPanelProps>(({children,...props},forwadedRef):React.ReactElement=>{
```

Comment:
What's by is there a <> here

Reply:
I don't understand sorry

Reply:
Why do you have <Div, props>
Shouldn't the props just extend Div?

Reply:
This is how it is done when you need to forward a ref in typescript

Reply:
But why do you need Div, Props
Can you just have Props
Then have props extend Div?

Reply:
HTMLDivElement refers to the ref type

Reply:
Then isn't the ordering incorrect?
Becuase the ref is the second parameter not the first

Reply:
Yes, that's right, but this is how it is. blame react! hehe

Reply:
Oh my god you are right haha this is so inconsistent with everything else, god damn React!!